src/legacy/core_plugins/metric_vis/public/metric_vis_fn.js

```diff
@@ -32,21 +32,114 @@ export const metric = () => ({
     defaultMessage: 'Metric visualization'
   }),
   args: {
-    visConfig: {
-      types: ['string', 'null'],
-      default: '"{}"',
+    percentage: {
+      types: ['boolean'],
+      default: false,
+      help: i18n.translate('metricVis.function.percentage.help', {
+        defaultMessage: 'Shows metric in percentage mode. Dont forget to set colorRange.'
```

Comment:
`Dont` -> `Don't`

Or would it be better to say `Requires colorRange to be set`?